packages/client/src/library/components/InputAddress.js

```diff
@@ -18,17 +25,19 @@ const InputAddress = ({ web3, value, isValid, onChange }) => {
   };
 
   return (
-    <div className="is-flex">
+    <div className={`is-flex ${className}`}>
       <div className="flex-1" style={{ position: "relative" }}>
         <input
-          style={{ height: 48 }}
-          className="border-light rounded-sm column is-full p-2 mt-2"
+          className="border-light rounded-sm column is-full p-4"
           type="text"
           value={value}
           onChange={onValueChange}
+          disabled={readOnly}
         />
         {isValid && (
-          <div style={{ position: "absolute", right: 17, top: 20 }}>
+          <div 
+            className="is-flex is-align-items-center" 
+            style={{ position: "absolute", right: 15, top: 0, height: "100%" }}>
```

Comment:
use a container with 100% height thats `align-items: center` which should be more flexible than fixed absolute positioning